openfisca_france/model/prestations/prestations_familiales/aeeh.py

```diff
@@ -65,16 +66,59 @@ def formula_2003_01_01(famille, period, parameters):
-    # complément de l'AEEH. Les parents d'enfants handicapés doivent donc choisir entre le versement
-    # du complément d'AEEH et la PCH.
+        # L'attribution de l'AEEH de base et de ses compléments éventuels ne fait pas obstacle au
+        # versement des prestations familiales.
+        # L'allocation de présence parentale peut être cumulée avec l'AEEH de base, mais pas avec son
+        # complément ni avec la majoration de parent isolé.
+        # Tous les éléments de la prestattion de compensation du handicap (PCH) sont également ouverts
+        # aux bénéficiaires de l'AEEH de base, sous certaines conditions, mais ce cumul est exclusif du
+        # complément de l'AEEH. Les parents d'enfants handicapés doivent donc choisir entre le versement
+        # du complément d'AEEH et la PCH.
 
         # Ces allocations ne sont pas soumis à la CRDS
         return montant_total
+
+
+class besoin_educatif_particulier(Variable):
+    value_type = bool
+    entity = Individu
+    label = "Enfant possède une reconnaissance d’un besoin éducatif particulier"
+    definition_period = MONTH
+
+
+class aeeh_eligible(Variable):
+    value_type = bool
+    entity = Famille
+    label = "Éligilité à l'Allocation d'éducation de l'enfant handicapé"
+    reference = "https://www.legifrance.gouv.fr/codes/section_lc/LEGITEXT000006073189/LEGISCTA000006156691/#LEGIARTI000006750709"
+    documentation = """
+                    Art. L. 541-1 à 4 du Code de la sécurité sociale (CSS), art. R. 541-1 à 10 du CSS, art. D. 541-1
+                    à 4 du CSS, arrêté du 24 avril 2002 relatif aux conditions d’attribution des six catégories de
+                    complément d’allocation d’éducation spéciale.
+                    """
+    definition_period = MONTH
+    set_input = set_input_divide_by_period
+    calculate_output = calculate_output_add
+
+    def formula_2020_01(famille, period, parameters):
+        janvier = period.this_year.first_month
+        age = famille.members('age', janvier)
+        taux_incapacite = famille.members('taux_incapacite', janvier)
+        besoin_educatif_particulier = famille.members('besoin_educatif_particulier', janvier)
+
+        prestations_familiales = parameters(period).prestations.prestations_familiales
+
+        residence = famille.members.menage('residence', period)
+
+        condition_age = (age < prestations_familiales.aeeh.age)
+        condition_taux_incapacite = ((
+            taux_incapacite >= prestations_familiales.aeeh.taux_incapacite_maximal.taux_incapacite_maximal_aeeh) + (
+            (
+                taux_incapacite >= prestations_familiales.aeeh.taux_incapacite_minimal.taux_incapacite_minimal_aeeh) * (
+                taux_incapacite < prestations_familiales.aeeh.taux_incapacite_maximal.taux_incapacite_maximal_aeeh) * besoin_educatif_particulier))
+
+        condition_residence_FR = False if residence == TypesLieuResidence.non_renseigne else True
```

Comment:
Appliquer une syntaxe vectorielle ?
```suggestion
        condition_residence_FR = not_(residence == TypesLieuResidence.non_renseigne)
```
en important `not_` en en-tête du fichier.

Reply:
@guillett Que conseillerais-tu comme variable pour signifier la résidence en France ?